spacialgaze-plugins/misc.js

```diff
@@ -433,4 +433,26 @@ exports.commands = {
 		Db.disabledScrolls.remove(target);
 	},
 	enableintroscrollhelp: ["/enableintroscroll [room] - Enables scroll bar preset in the room's roomintro."],
+
+	'!ai': true,
+	askai: 'ai',
+	'8ball': 'ai',
+	ai: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		const http = require('http');
```

Comment:
* if u declare a variable on top of file
* its usually done for multiple usages of a single variable
* why will i do it on the top, if its needed only once ?